src/components/PersonInformationForm/PersonInformationForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import { Form } from 'react-bootstrap'
+import { FormWrapper } from './PersonInformationForm.style'
+
+const PersonInformationForm = () => {
+  const [personInfo, setPersonInfo] = useState({
+    gender: 'Male',
+    age: 25,
+    height: 175,
+    weight: 70,
+  })
+
+  const genders = ['Male', 'Female']
+
+  const handleOnlyNumbers = (input) => {
```

Comment:
Can you change the `input` to be called `event` please? It's a convention

Reply:
Fixed